cli/cli.go

```diff
@@ -30,35 +28,33 @@ import (
 	"github.com/mendersoftware/mender/installer"
 	"github.com/mendersoftware/mender/system"
 
+	"github.com/alfrunes/cli"
```

Comment:
Would it not be easier to create a `mendersoftware` repo. And have this as a PR to master there. Review the new CLI lib standalone, and then take it from there, instead of this all together in one big hunk?

Reply:
It might be, but it's still quite a lot of work, so let's make sure it's worth it, and this is what we want. For example, another option we might consider is whether it's easy to strip out the template dependency that @alfrunes mentioned from the original urfave/cli to shrink it. If that is 10% of the work, maybe we should consider it.

For one thing, doing this entire PR is *way* more work than the 5SP allocated to this task, even if starting from where we are now.

Reply:
Fair point. Let's move the discussion back to the ticket then?

Reply:
Another advantage of that is that we can use the same library everywhere. Assuming it's possible, I would favor this option over rolling our own library.

Reply:
> Let's move the discussion back to the ticket then?

But yes, let's!